casesC.Rmd

```diff
@@ -0,0 +1,71 @@
+---
+title: "casesC"
+author: "cazeredotseng"
+date: "7/20/2020"
+output: html_document
+---
+
+## R Markdown
+
+Setting work directory and separating variables from data file us-states.csv
+
+```{r setup}
+setwd("C:\\Users\\Cassandra\\Documents\\SummerR\\HW03")
+#install.packages("ggplot2")
+library(ggplot2)
+#install.packages("viridis")  # Install
+library("viridis")           # Load
+
+us_states <- read.csv(file="us-states.csv", head=TRUE, sep=",")
+
+state  <- us_states$state
+cases  <- us_states$cases
+date   <- us_states$date
+deaths <- us_states$deaths
+percent <- deaths/cases
+#comb    <- cbind(date, cases, state)
+
+```
+
+## Plot
+
+Plots for covid cases per state:
+
+```{r casedata, echo=FALSE}
+
+# ggplot(us_states, aes(date,cases,color=state,group=state))+
+#   geom_point()+
+#   scale_y_continuous( "Covid Cases (thousand)",
+#     breaks = c(0e+00, 1e+05, 2e+05, 3e+05, 4e+05),
+#     labels = c('0', '10', '20', '30', '40')
+#   )
+
+
+ggplot(us_states, aes(x=state, y=cases, fill=percent*100))+
+  geom_bar(
+    stat="identity",
+    position=position_dodge(),
+    color="black"
+  ) +
+  theme(
+    plot.title=element_text(hjust=0.5),
+    axis.text.x = element_text(angle = 90, size=10),
+    panel.grid.major=element_blank()
+  )+
+  scale_y_continuous( 
+    "Cases (1e5)",
+    breaks = c(0e+00, 1e+05, 2e+05, 3e+05, 4e+05),
+    labels = c('0', '1', '2', '3', '4')
+  )+
+  labs(x = "", fill = "Deaths/Cases %")+
+  scale_fill_gradient(low = "white", high = "blue")+
+  ggtitle("Covid Cases per State (1/21-7/19)")
+
```

Comment:
Overall your code looks good. I was able to clone the project and it ran smoothly on my end. The goal was clear and the code is clean to follow. The plot itself was also a pretty unique way of presenting the covid data; I liked the addition of the color fill for deaths/cases % to make a more informative figure. 

There are only a couple things I would suggest for changes:
1. I think the y-axis scale being the true value instead of having the 1e5 axis label would be a little simpler, but that's mostly just my personal preference.
2. The centered text for the x-axis labels isn't the easiest to use when aligning which state corresponds to its respective bar but I honestly can't figure out a better solution. Changing the angle argument adds overlap and vjust doesn't do much to help either. If the last letter from each state was directly adjacent to its tick marker then that would probably help, but I can't figure out how to customize the text labels for the state names that way. 
3. Maybe the inclusion of an additional subplot for the states that had a low number of total covid cases? The information added from the deaths/cases % fill is lost in low case states when they're on the same scale as the states with significantly more cases.